### DISCUSSION TITLE
If network configuration of client changes (e.g. Wifi to LAN) we may get audio issues for a period of time

### DISCUSSION THREAD

**Discussion by corrados** [jamulussoftware/jamulus#1082] [Bug Investigation]
See https://github.com/corrados/jamulus/issues/396#issuecomment-653683410

**Comment by pljones** [comment]
This is tricky to address.

The UDP nature of Jamulus means it's connectionless.  So it doesn't "know" (i.e. the IP layer doesn't tell it) "your connection went down" when someone "unplugs" (virtually) the network connection.  It just keeps trying.

I'm wondering if the packet numbering may have helped with this.

**Comment by gilgongo** [comment]
If we can nail down what needs to be fixed to address this then it can be a ticket, and if it needs more discussion I suggest that could take place here (and not in Discussions unless we want to have a new category of "Bug Investigations"?).

If you do discuss here, please edit the ticket to reflect the latest thinking rather than leaving thoughts and findings scattered about the thread. Otherwise feel free to convert to discussion and we may need to create that category to round up similar in future?]

**EDIT**: I have in fact now created a "Bug Investigations" category for some other stuff. So moving there if that's OK.
